File: txt_File/Python/filetxt.py
# Undergraduate Student: Arturo Burgos
# Professor: João Rodrigo Andrade
# Federal University of Uberlândia - UFU, Fluid Mechanics Laboratory - MFLab, Block 5P, Uberlândia, MG, Brazil


# Fifth exercise: txt File 



import numpy  
import time

x = numpy.arange(1,1000000,1)
X = str(x)
#print(x)

y = numpy.cos(x)

#print(y)




def writetxt(a1,a2):

    z = open('/home/arturo/Desktop/GitHub/IC/Comparing-Languages/txt_File/Python/teste.txt',"w")
    z.write("X")
    z.write('\n')

    for i in range(len(a1)):
    
        z.write(str(a1[i]))
        z.write(' ')

    z.write('\n')
    z.write("Y")
    z.write('\n')

    for i in range(len(a2)):
    
        z.write(str(a2[i]))
        z.write(' ')

    z.close()


t_initial = time.time()
writetxt(x,y)

print("--- %s seconds ---\n" % (time.time() - t_initial))

File: txt_File/Julia/filetxt.jl
# Undergraduate Student: Arturo Burgos
# Professor: João Rodrigo Andrade
# Federal University of Uberlândia - UFU, Fluid Mechanics Laboratory - MFLab, Block 5P, Uberlândia, MG, Brazil


# Fifth exercise: txt File 


using Printf # Just the same as import a module
using Statistics
using CPUTime

x = collect(1:1:10000)

y = cos.(x)

#println(x)

#println(y)

t1 = time_ns()
@elapsed open("teste.txt","w") do io

    for i in x
        println(io,i)
    end

    for i in y
        println(io,i)
    end
    
end
t2 = time_ns()
timeo = (t2-t1)/1.0e9

println(timeo)



